app/packages/spaces/src/components/Space.tsx

```diff
@@ -80,8 +80,12 @@ export default function Space({ node, id }: SpaceProps) {
 
     return (
       <PanelContainer>
-        {node.isRoot() && <Workspaces />}
-        <PanelTabs data-type="panel-container" data-cy="panel-container">
+        {node.isRoot() && archetype !== "modal" && <Workspaces />}
```

Comment:
Not requesting any action, but we can probably refactor this. This was misplaced/rushed by me